particle/particle/particle.py

```diff
@@ -388,39 +399,46 @@ def load_table(cls, filename=None, append=False):
         else:
             cls._table_names.append("{0!r} {1}".format(filename, len(cls._table_names)))
 
-        with filename as f:
+        # The following 2 files are necessary for Python 2, otherwise simply
+        # with filename as f:
+        # works just fine in Python 3 !
+        from contextlib import closing
+
+        with closing(filename) as f:
```

Comment:
I assume closing on something that is already a context manager is fine?

Reply:
I found this workaround online and it seems it is precisely to solve thie kind of issue I got with Python 2 here.